modules/rhel-adding-node.adoc

```diff
@@ -43,10 +43,17 @@ fully-qualified domain name for each host. This name is the host name that the
 cluster uses to access the machine, so set the correct public or private name
 to access the machine.
 
-. Run the playbook:
+. Navigate to the Ansible playbook directory:
```

Comment:
Do we want a [source,terminal] after Step 1 (_Create an Ansible inventory file that is named /<path>/inventory/hosts_)?

Reply:
Since an Ansible inventory file is kind of a wildcard when it comes to format, I opted to leave it as generic source.